BaseLib/ConfigTree.h

```diff
@@ -0,0 +1,31 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2015, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#ifndef BASELIB_CONFIGTREE_H_
+#define BASELIB_CONFIGTREE_H_
+
+#include <string>
+#include <boost/property_tree/ptree.hpp>
```

Comment:
There is also the header file `boost/property_tree/ptree_fwd.hpp`, which only has some forward declarations (?).
Maybe it's better to use that one here.